apps/web/src/state/index.tsx

```diff
@@ -4,12 +4,11 @@ import { TwilioError } from 'twilio-video';
 import { settingsReducer, initialSettings, Settings, SettingsAction } from './settings/settingsReducer';
 import useActiveSinkId from './useActiveSinkId/useActiveSinkId';
 import usePasscodeAuth from './usePasscodeAuth/usePasscodeAuth';
-import { User } from 'firebase';
 
 export interface StateContextType {
   error: TwilioError | Error | null;
   setError(error: TwilioError | Error | null): void;
-  user?: User | null | { displayName: undefined; photoURL: undefined; passcode?: string };
+  user?: null | { displayName: string; photoURL: string; passcode?: string };
```

Comment:
```suggestion
  user?: null | { passcode?: string };
```

Reply:
Along with this, we can delete the UserAvatar component since it's only used to display the Firebase user info.